content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,87 @@
+## Installation
+
+- **Added** experimental [manifest and profile commands](docs/setup/install/operator/) to install and manage the Istio control plane for evaluation.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. You no longer need to explicitly name service ports as `http` or `tcp`.
+- **Added** a mode to the Gateway API for mutual TLS operation. The new mode can read certificates via SDS.
+- **Added** Envoy proxy readiness checks. They now check Envoy's readiness status.
+- **Fixed** issues present when a service communicates over the network first in permissive mutual TLS mode for protocols like MySQL and MongoDB.
+- **Improved** container ports are no longer required in the pod spec. All ports are [captured by default](/faq/traffic-management/#controlling-inbound-ports).
+- **Improved** the `EnvoyFilter` API. You can now add or update all configurations.
+- **Improved** the Redis load balancer to now default to [`MAGLEV`](https://www.envoyproxy.io/docs/envoy/v1.6.0/intro/arch_overview/load_balancing#maglev) when using the Redis proxy.
+- **Improved** load balancing to direct traffic to the [same region and zone](docs/ops/traffic-management/locality-load-balancing/) by default.
+- **Improved** Pilot performance. The reduction in CPU utilization could be up to 10 times depending on deployment.
```

Comment:
** Improved ** Pilot cpu utilization has been reduced by up to 90% depending on the specific deployment.

Reply:
this doesn't make sense as a sentence?

Reply:
here is what i went with: 
- **Improved** Pilot by reducing CPU utilization. The reduction approaches 90% depending on the specific deployment.

Please click `Resolve conversation` if you are satisfied with this change, otherwise, please offer an alternative.